Task 5 - экспорт фигур

SergoShe opened the pull request:
Необходимо дать возможность пользователю экспортировать свои фигуры. На выбор формат json или xml. Схему придумать самостоятельно.

SergoShe committed:
Task 5 - first view

rs-pluss commented on src/Rectangle.java:
```diff
@@ -15,8 +16,8 @@ public String getName() {
     @Override
-    public String getType() {
-        return "Rectangle";
+    public InputShapeType getType() {
```
Это внешний enum, если используешь его внутри то надо переименовать

rs-pluss commented on src/JSONWorker.java:
```diff
@@ -0,0 +1,23 @@
+import java.util.HashMap;
+import java.util.LinkedHashMap;
+
+public class JSONWorker {
```
JsonBuilder

rs-pluss commented on src/Geosandbox.java:
```diff
@@ -93,4 +93,24 @@ private void showShape(HashMap<String, Shape> shapeList) throws IOException {
+        System.out.println("Enter file format:");
+        System.out.println("1.JSON\n2.XML");
+        String enter = reader.readLine();
+        if (enter.equals("1")) {
```
Enum

SergoShe committed:
- Rename:
 InputShapeType -> ExternalShapeType
 JSONWorker -> JsonBuilder
 XMLWorker -> XMLBuilder
- Add enum FileType
